### PR TITLE
imports 'events' into __init__.py

### LINKED ISSUE #305: Make the events module available in __init__.py
This is a quality of life thing. Currently, if I write code like this:

    import ppb

    class Player(ppb.BaseSprite):
        def on_update(self, update: ppb.events.Update, signal):
            pass

PyCharm complains "Cannot find reference 'events' in '__init__.py'"

We can fix this by importing it explicitly in the init file.

### PR DESCRIPTION
**Pull request by BubbsTheSupreme**
This resolves #305. I used VS Code to test.

### CODE REVIEW

**Comment by BubbsTheSupreme** [comment]
If you don't mind me asking, what is `__all__` referring to? Is it a file or multiple files? I'm new to this project (and contributing to open source) so I'm not quite sure.

**Comment by astronouth7303** [comment]
https://github.com/ppb/pursuedpybear/pull/334/files#diff-4f4e1b10744df4e400dcb41147d6df6dR11-R16

**Comment by astronouth7303** [comment]
Also, please look at CI.

File: ppb/__init__.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import logging
+import events
```
Comment (pathunstrom) [inline ppb/__init__.py]:
```suggestion
```

File: ppb/__init__.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 from typing import Callable
```
Comment (pathunstrom) [inline ppb/__init__.py]:
```suggestion

from ppb import events
```

**Comment by pathunstrom** [comment]
@BubbsTheSupreme `__all__` defines exported names for usage in `from ppb import *`. You should add to the shortcuts line the string `"events"` to get what @astronouth7303 is asking for.

File: ppb/__init__.py (hunk 2 above)
Comment (BubbsTheSupreme) [reply ppb/__init__.py]: Should I change the `import events` to `from ppb import events` ?

Comment (BubbsTheSupreme) [reply ppb/__init__.py]: I tried this and it passed the 9 tests!

**Pull request merged**